scripts/globals/mobskills/Medusa_Javelin.lua

```diff
@@ -16,16 +16,23 @@ end;
 
 function onMobWeaponSkill(target, mob, skill)
 
+	local mobID = mob:getID();
 	local numhits = 1;
 	local accmod = 1;
 	local dmgmod = 1.5;
+	if (mobID == 16912876) then 
+   	dmgmod = 10.5;
+    	end
 	local info = MobPhysicalMove(mob,target,skill,numhits,accmod,dmgmod,TP_NO_EFFECT);
 	local dmg = MobFinalAdjustments(info.dmg,mob,skill,target,MOBSKILL_PHYSICAL,MOBPARAM_NONE,info.hitslanded);
-	target:delHP(dmg);
+	target:delHP(dmg);	

```

Comment:
Trailing tab, and extra whitespace near the return below, kill with fire pretty please. :smile: